lib/stripe/stripe_object.rb

```diff
@@ -428,15 +420,14 @@ def dirty_value!(value)
     # StripeObject.
     def empty_values(obj)
       values = case obj
-      when Hash         then obj
-      when StripeObject then obj.instance_variable_get(:@values)
-      else
-        raise ArgumentError, "#empty_values got unexpected object type: #{obj.class.name}"
+               when Hash         then obj
```

Comment:
Not a big fan of this indent style.

Reply:
Yeah I don't like this, the trouble is at any point you rename `values` then every single line has to change.

Reply:
Switched `Layout/CaseIndentation` to `end` to solve this.